EventListener/ViewResponseListener.php

```diff
@@ -87,14 +87,10 @@ public function onKernelView(GetResponseForControllerResultEvent $event)
                 $view->setStatusCode($configuration->getStatusCode());
             }
             if ($configuration->getSerializerGroups() && !$customViewDefined) {
-                $context = $view->getSerializationContext() ?: new SerializationContext();
-                $context->setGroups($configuration->getSerializerGroups());
-                $view->setSerializationContext($context);
+                $context->addGroups($configuration->getSerializerGroups());
             }
             if ($configuration->getSerializerEnableMaxDepthChecks()) {
-                $context = $view->getSerializationContext() ?: new SerializationContext();
-                $context->enableMaxDepthChecks();
-                $view->setSerializationContext($context);
+                $context->setMaxDepth(0);
```

Comment:
Yes, for a serialization context it does the same thing :
```php
        if ($context instanceof MaxDepthContextInterface && null !== $context->getMaxDepth()) {
            $newContext->enableMaxDepthChecks();
        }
```

But for a deserialization context it does that in addition :
```php
        if ($context instanceof MaxDepthContextInterface && null !== $context->getMaxDepth()) {
            for ($i = 0; $i < $context->getMaxDepth(); $i++) {
                $newContext->increaseDepth();
            }
        }
```

I have to check if this is really what we want.